core/templates/components/brand-bar/brand-bar.twig

```diff
@@ -0,0 +1,19 @@
+{#
+/**
+ * @file
+ * Brand Bar Component.
+ *
+ * Stanford brand bar with the wordmark logo with four variants:
+ *   - Default: White wordmark over cardinal red background
+ *   - Bright: White wordmark over bright red background
+ *   - Dark: White wordmark over dark grey background
+ *   - White: Cardinal red wordmark over white background
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - modifier_class: Additional css classes to change look and behaviour.
+ */
+#}
+<section id="su-brand-bar" class="su-brand-bar {{ modifier_class }}">
```

Comment:
Homesite currently has a `<section id="brand-bar">`, but that section contains several items:

- The wordmark
- The gateway nav
- The menu toggle (mobile)
- The search toggle (mobile)

Looking at the candidate patterns for the masthead, it looks like the brand bar will only ever contain the wordmark, even on mobile. Does it therefore need to be wrapped in a `<section>`. Could we just use the `<div>`? I anticipate there will be a `<section>` for the masthead. Maybe we don't need to nest an additional section that only contains the word mark?

Reply:
I think the <section> tags are needed (or at least an outside container tag pair) to work with Richard's CSS grid mixin. The mixin takes the first element inside that parent outside container and stick it in the 2 column of the grid. In the beginning, I didn't have the <div> container around the <a> element, and it ended up making the whole brand bar clickable (the whole center container portion) because it sticks the <a> element inside the 2nd grid column and it takes up the full width of that column.

Reply:
Ah. I see. Thanks!

You may end up refactoring this when you work on the masthead, then. I suspect the masthead will end up in a `<section id="masthead">`, and maybe that will `@include centered-grid-container`. We'll see what makes sense then.

Reply:
Yeah, I can definitely see some refactoring needed.